subprojects/core-api/src/main/java/org/gradle/vcs/VersionRef.java

```diff
@@ -19,21 +19,12 @@
 import org.gradle.api.Incubating;
 
 /**
```

Comment:
I think you've reverted some of the changes from master to this class.

Reply:
This was a mistake, putting it back.